Update sensors.md to include BLE transmitter. (#449)
home-assistant/companion.home-assistant: docs/core/sensors.md

| `is_charging` | Whether or not the device is actively charging |

## Bluetooth Sensors
![Android](/assets/android.svg)<br />

> This Bluethoot Connection state will be the total number of connected bluetooth devices. The sensor will update as soon as the bluetooth state of the device changes. This sensor makes use of Android's [Bluetooth](https://developer.android.com/reference/android/bluetooth/package-summary?hl=en) package.

Reviewer: ```suggestion
The Bluetooth Connection state will be the total number of connected bluetooth devices. The sensor will update as soon as the bluetooth state of the device changes. This sensor makes use of Android's [Bluetooth](https://developer.android.com/reference/android/bluetooth/package-summary?hl=en) package.
```

Reviewer: Bluetooth is still misspelled :)

| Attribute | Description |
| --------- | --------- |

>  `Connected Paired Devices` | The list of paired devices that are currently connected. |

Reviewer: ```suggestion
| `Connected Paired Devices` | The list of paired devices that are currently connected. |
```

looks like this was removed on accident

Reviewer: Looks like the suggestion is still missing?

| `Connected Not Paired Devices` | The list of devices that are connected but not paired. |
| `Paired Devices` | The list of devices that are paired. |

There will also be a binary sensor for the `bluetooth_state` that will represent whether or not bluetooth is turned on for the device. This sensor will update anytime the state of bluetooth changes.

> A BLE Transmitter sesnor allows your device to transmit a BLE iBeacon.  This is useful in conjunciton with projects like roomassistant and esp32-mqtt-room to allow room level tracking.

Reviewer: ```suggestion
A BLE Transmitter sensor allows your device to transmit a BLE iBeacon.  This is useful in conjunction with projects like roomassistant and esp32-mqtt-room to allow room level tracking.
```

Reviewer: Lets also make sure to tell users how often this emits and how to get the beacon ID etc...  Users will be referring to this page for the real details on things work since the sensor description in app can only be so long and helpful.

Reviewer: Also we need to mention how the sensor settings work
